docs/eigenda/integrations-guides/rollup-guides/op-stack/deployment.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Public Deployment
+sidebar_position: 2
+---
+
+# Public Deployment
+
+This guide will show you how to deploy an OP Stack rollup on a public Ethereum
+network, using an EigenDA proxy instance for secure communication with DA. 
```

Comment:
* Would be good to mention the plasma spec someplace in this sentence.
* "secure communication with the DA network"

Reply:
[c848a2b](https://github.com/Layr-Labs/eigenlayer-docs/pull/257/commits/c848a2b082478e036c48fe607283eeb598782102)